### ISSUE TITLE
TalkBack conflicts with TTS

### ISSUE THREAD

**Issue by david-allison-1** [ankidroid/Anki-Android#6369]
TTS doesn't work if TalkBack is enabled, this leads to a poor accessibility experience for partially-sighted users.

This one probably needs a lot of upfront design to get working nicely. Likely want to get a few opinions from affected users.

**Comment by Gambit-Checkmate** [comment]
Hi! I'd like to help with this, but I want to start with the behavior rather than jump into code. I can reproduce the TalkBack/TTS interaction on a test device, collect the exact navigation/announcement flow, and then propose a small fix or a focused follow-up issue. Is this still a useful direction, and are there any current TTS/TalkBack constraints I should know about?

**Comment by david-allison** [comment]
@Gambit-Checkmate 

Research here would be fantastic! No current project-level constraints. I'm more familiar with the Android Accessibility APIs these days, but haven't had the capacity to apply this knowledge to AnkiDroid.

For the implementation: I'd want to be much more test-driven with accessibility work, to be sure we don't introduce regressions.
